docs/Advanced-Topics-Block-Components.md

```diff
@@ -76,6 +76,28 @@ If your component contains text as provided by your `ContentState`, your custom
 component should compose a `DraftEditorBlock` component. This will allow the
 Draft framework to properly maintain cursor behavior within your contents.
 
+`DraftEditorBlock` is exposed as `EditorBlock` in the core library. By default it must have props
```

Comment:
Throughout this content, this should refer to `EditorBlock` instead of `DraftEditorBlock`, since that's how it is exposed. Then you don't need to qualify the renaming with this sentence.